docs/rfcs/0000-namespace.md

```diff
@@ -0,0 +1,76 @@
+- Start Date: (2015-10-29)
+- RFC PR: (leave this empty)
+- AMS Issue: (leave this empty)
+
+# Summary
+
+Provide a consistent API for the user of the AMS.
+
+# Motivation
+
+The actual public API is defined under `ActiveModelSerializers`,
+`ActiveModel::Serializer` and `ActiveModel`.
+
+At the `ActiveModel::Serializer` we have:
+
+- `ActiveModel::Serializer.config`
+- `ActiveModel::Serializer`
+
+At the `ActiveModelSerializers` we have:
+
+- `ActiveModelSerializers::Model`
+- `ActiveModelSerializers.logger`
+
+At `ActiveModel` we have:
+
+- `ActiveModel::SerializableResource`
+
+The idea here is to provide a single namespace `ActiveModel::Serializers` to the user.
+Following the same idea we have on other gems like
+[Devise](https://github.com/plataformatec/devise/blob/e9c82472ffe7c43a448945f77e034a0e47dde0bb/lib/devise.rb),
+[Refile](https://github.com/refile/refile/blob/6b24c293d044862dafbf1bfa4606672a64903aa2/lib/refile.rb) and
+[Active Job](https://github.com/rails/rails/blob/30bacc26f8f258b39e12f63fe52389a968d9c1ea/activejob/lib/active_job.rb)
+for example.
+
+# Detailed design
+
+We are adding a extension for the Active Model, so
+[following the Rubygens recomendation](http://guides.rubygems.org/name-your-gem/)
+for the gem name we need to change to this.
+
+|Gem name                  | Require statement          | Main class or module     |
+|--------------------------|----------------------------|--------------------------|
+| ActiveModel::Serializers | `active_model/serializers` | ActiveModel::Serializers |
```

Comment:
@bf4 I know that it is will be a big change, that will hurt. I added some of the pains on the drawbacks sections, could you please expand your idea of the pains.

Even with the pain I think this is a good time to do that, since the AMS does not reach 1.0, if we will follow [Semantic Version](http://semver.org/) it is the right time to break things.

Thanks for the feedback